### PR TITLE
Wishlist management page

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**
Added wishlist management page

### CODE REVIEW

File: cartManagement.html [hunk 1]
```diff
@@ -78,14 +78,14 @@ <h2 class="cart_page_title">My Cart (2)</h2>
             <div class="Card-horizontalImage-text">
               <img
                 class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce cart_card_imgresize"
-                src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+                src="https://d3cif2hu95s88v.cloudfront.net/blog/wp-content/uploads/2021/05/04150624/Spider-plant.jpg"
```
Comment (VLeads) [inline cartManagement.html]: also add alt tag to images.

File: cartManagement.html [hunk 2]
```diff
@@ -116,14 +118,14 @@ <h3 class="card-heading">Nike Men's Flex 2020</h3>
             <div class="Card-horizontalImage-text">
               <img
                 class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce cart_card_imgresize"
-                src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+                src="https://pyxis.nymag.com/v1/imgs/4a7/29b/4ef2f518834dd81d8ceb374caa54fd5171-22-plant-gifts.rsquare.w700.jpg"
```
Comment (VLeads) [inline cartManagement.html]: You can add loading="lazy" to images. It will help in performance of the app.

**Review by VLeads — APPROVED** [review]
There are some improvement suggestions, else code is good !

**Comment by VLeads** [comment]
Try to make only few commits. Large number of commit make review time taking

**Review by MayankKumar10 — APPROVED** [review]
Look into Vishal Kumar Suggestions, everything else looks good

File: cartManagement.html (hunk 1 above)
Comment (abhisheksahani-nova) [reply cartManagement.html]: Got it

File: cartManagement.html (hunk 2 above)
Comment (abhisheksahani-nova) [reply cartManagement.html]: Yess I will do that

**Comment by abhisheksahani-nova** [comment]
> Try to make only few commits. Large number of commit make review time taking

Okay, I will try to do that

File: cartManagement.html (hunk 1 above)
Comment (abhisheksahani-nova) [reply cartManagement.html]: It has already been added, yu might be seeing older commits

**Pull request merged**